src/hevm/hevm.cabal

```diff
@@ -1,3 +1,4 @@
+cabal-version: 2.2
```

Comment:
What about 2.2 specifically? And why no >=?

Reply:
see 27dbc97c0bbba34907c115abf5e9cea4821fa019. We could probably revert the cabal file to how it was, leaving just the `libff` related changes.